gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinClusterManager.java

```diff
@@ -218,6 +220,7 @@ private void startAppLauncherAndServices() {
     }
 
     this.applicationLauncher.start();
+    this.started = true;
```

Comment:
Does it makes sense to set it at the end of the "start" method? It's not obvious that this function will be called last on the start, and this sequence can change in the future.